### ISSUE TITLE
Multi-select tracks

### ISSUE THREAD

**Issue by timoncool** [Zoobumafu/Playlist-Saber#6]
Hey! This is I again.
Today I convert other 200+ files from my Spotify library and generate maps for them. 
I open playlistsaber, create a new playlist, sort songs by date, and go to be mad)) I need 200+ double clicks to add new songs to my new playlist.
Please add multi-select functionality, like shift + arrows, or cursor multi-select, as it works in windows. 
I really love Playlistsaber, but some things need to improve.

I will be glad to give u other advice for better UX and functionality if u wish =)

**Comment by Zoobumafu** [comment]
Hey @timoncool !,
I am aware of this problem, and I previously tried to solve it, only to realize it is not supported by the framework I use, I will try to add something like that sooner or later, but I doubt if I can, so no promisses about it.

BUT - What you can do, is mark "Show only songs that are not in playlist" - which I assume would be only the newly downloaded songs, and than click on the button "Add songs below to selected playlist" - it is below the "Save changes button", it only adds songs that are in your current view (in this game, only songs that are not in playlist).

And I am open for UX advice, as the dev (who has been looking at the software for 1000 times already), I am having hard time understanding how the end user sees the program, and how simple it is to them.

**Comment by timoncool** [comment]
Hey @Zoobumafu! Maybe u can use something like a checkbox for mass selection? I mean u can select one first in list and one on end and select all between the first and end tracks. It's a little bit strange, but maybe useful. 

Yep, I know about Show only songs that are not in playlist and Add songs below to the selected playlist, but I create a new one and this is not useful for my task.

About a little improvement: add some info for the tracklist, like the artist, mapper, and added date, because if u have a big tracks library u have many tracks with the same names, and looking need one is difficult. Maybe show info about bpm and about the difficulty level? With sorting for any of those parameters.

**Comment by Zoobumafu** [comment]
@timoncool I can add checkboxes and work with them, I will see if there's an option to select all between, I believe it should be possible, just need to see how the button would fit on the already crowded form..

About the track info, it was already there in the "more info" panel, I added the date modified (or added) label now.

![image](https://user-images.githubusercontent.com/81674290/155478197-ebf6e653-9de5-493c-8b5f-0269e691bb2e.png)

**Comment by timoncool** [comment]
> About the track info, it was already there in the "more info" panel, I added the date modified (or added) label now.

Yep, it good, but more useful if this info show on tracklist  =)

**Comment by Zoobumafu** [comment]
> > About the track info, it was already there in the "more info" panel, I added the date modified (or added) label now.
> 
> Yep, it good, but more useful if this info show on tracklist =)

I know, but the form is way too crowded as it is.. I'll see if it can fit somewhere..
